rasa_core/evaluate.py

```diff
@@ -403,14 +369,22 @@ def _predict_tracker_actions(tracker, agent, fail_on_prediction_errors=False,
                                                        events[:1],
                                                        agent.domain.slots)
 
+    tracker_actions = []
+
     for event in events[1:]:
         if isinstance(event, ActionExecuted):
-            action_executed_result, policy = \
+            action_executed_result, policy, confidence = \
                 _collect_action_executed_predictions(
                         processor, partial_tracker, event,
                         fail_on_prediction_errors
                 )
             tracker_eval_store.merge_store(action_executed_result)
+            tracker_actions.append(
+                    {"action": action_executed_result.action_targets[0],
```

Comment:
can this never ever be empty?

Reply:
I don't believe so, because `processor.predict_next_action()` always predicts _something_